main.go

```diff
@@ -1,3 +1,12 @@
+// This is a TB patched version. The extra feature is to merge toEmail lists.
+// Usage:
+//   Add annotation "sensu.io/plugins/email/config/toEmail/merge" to check and/or entity.
```

Comment:
for the annotation, consider using `sensu.io/plugins/email/config/toEmail/addresses` instead